src/Microsoft.Azure.SignalR.Management/ServiceManager.cs

```diff
@@ -23,33 +22,25 @@ internal class ServiceManager : IServiceManager
         private readonly IReadOnlyCollection<ServiceDescriptor> _services;
         private readonly ServiceEndpoint _endpoint;
         private readonly IServiceEndpointProvider _endpointProvider;
-        private readonly ServiceTransportType _transportType;
 
-        public ServiceManager(IReadOnlyCollection<ServiceDescriptor> services, IServiceProvider serviceProvider, RestClientFactory restClientFactory, IServiceEndpointManager endpointManager, IOptions<ServiceManagerOptions> options)
+        public ServiceManager(IReadOnlyCollection<ServiceDescriptor> services, IServiceProvider serviceProvider, RestClientFactory restClientFactory, IServiceEndpointManager endpointManager)
         {
             _services = services;
             _serviceProvider = serviceProvider;
             _restClientFactory = restClientFactory;
             _endpoint = endpointManager.Endpoints.Keys.First();
             _endpointProvider = endpointManager.GetEndpointProvider(_endpoint);
-            _transportType = options.Value.ServiceTransportType;
```

Comment:
what if cx set ServiceTransportType to Transient, will throw exception or print warning log?

Reply:
looks like the ServiceTransportType is useless in the future? plan to remove it?

Reply:
`ServiceTransport` is not useless. The behavior is not changed. Just some implementation changes.

Reply:
I see, moved to builder